rsconf/package_data/sirepo_sandbox/sshd_config.jinja

```diff
@@ -0,0 +1,10 @@
+AuthorizedKeysFile {{ sirepo_sandbox.guest.identity_pub_f }}
+HostKey {{ sirepo_sandbox.guest.host_key_f }}
+ListenAddress {{ sirepo_sandbox.ip }}:{{ sirepo_sandbox.ssh_port }}
+PasswordAuthentication no
+PermitTunnel no
+PermitUserEnvironment no
+PrintLastLog no
+PrintMotd no
+Protocol 2
+UseDNS no
```

Comment:
I don't think we need this. People will be logging in from real computers on the internet.

Reply:
566ec33